pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -143,5 +143,16 @@ let
       });
       xcbuild = xcodebuild;
     }));
+
+    darwin-stubs = stdenvNoCC.mkDerivation {
+      pname = "darwin-stubs";
+      version = "11.0";
```

Comment:
shouldn't this come from `MacOSX-SDK` ?

Reply:
It does a few lines down where it symlinks the required paths.

Reply:
sorry -- i ment to highlight version. eg: `inherit (MacOSX-SDK) version;`

Reply:
Done.